### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by nztvar** [httpcats/http.cat#136]
I shared the url with community devs and got immediate pushback from those who use screen readers to navigate the internet.

* Images do not have useful alt text
* Status code not in body in text; no indication of page purpose.

I believe the pages do not meet minimal [Web Content Accessibility Guide 2.0](https://www.w3.org/WAI/standards-guidelines/wcag/), but should be able to to get there very easily. It would also prepare the project for automated language translation.

**Comment by rogeriopvl** [comment]
Hi @nztvar ,

Thanks for the feedback. I'm currently out of time to address these issues. But please feel free to open a pull request.
